src/workflows-app/RunDetails.js

```diff
@@ -257,7 +240,13 @@ export const BaseRunDetails = (
       ],
       [
         workflow === undefined,
-        () => h(Fragment, [div({ style: { fontStyle: 'italic', marginBottom: '1rem' } }, ['Fetching workflow metadata...']), centeredSpinner()]),
+        div({ style: { width: '100%' } }, [
+          div({ style: { padding: '1rem 2rem 2rem' } }, [header]),
```

Comment:
If this is serving as a heading, an `h1-6` element would be more semantically meaningful.
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements
- https://web.dev/learn/html/semantic-html

Note, the actual heading level needs to fit in the page's hierarchy:
- https://www.a11yproject.com/posts/how-to-accessible-heading-structure/